### DISCUSSION TITLE
Unable to start streaming in WSL

### DISCUSSION THREAD

**Discussion by deliaam** [OvenMediaLabs/OvenMediaEngine#1135] [Help]
I have installed OME in my wsl. I have not changed anything in Server.xml. I am trying to publish with RTMP in OBS, but when I "Start Streaming" it says "Failed to connect to server, The connection timed out. Make sure you've configured a valid streaming service and no firewall is blocking the connection". Firewall is not enabled in wsl nor windows. 

sudo systemctl status ovenmediaengine shows: 
![image](https://user-images.githubusercontent.com/72461334/224984952-94660fb9-798d-41c7-bfc6-5264317f1ebd.png)

This is the stream configuration in obs: 
![image](https://user-images.githubusercontent.com/72461334/224985293-f2665e8a-f6f9-4352-8c2c-1ba5102a425e.png)

It doesn't seem to be logging anything. This is the log file: 
[ovenmediaengine.log](https://github.com/AirenSoft/OvenMediaEngine/files/10967626/ovenmediaengine.log)

What am I doing wrong?

**Comment by getroot** [comment]
There is nothing to be expected except that this is a firewall problem or you have mistyped the IP address. Because the connection doesn't even reach the OvenMediaEngine.

What is installed in WSL? Is it Ubuntu? You need to check if the firewall is running on Ubuntu. 

```
$ sudo ufw status verbose
Status: inactive
```

**Reply by deliaam** [reply]
It is Ubuntu. Firewall is not running

**Comment by getroot** [comment]
Since I don't use WSL, I cannot reproduce this problem. But I was able to find a similar problem and solution to yours on google. Hope this helps you. https://github.com/microsoft/WSL/issues/5018

**Reply by deliaam** [reply]
I did:

> 
> 1. run the following command in Powershell
>     Get-Service LxssManager | Restart-Service
> 2. Enabled the following Under "Turn Windows features on and off"
>     Hyper-V
>     Windows Hypervisor Platform
> 3. Restart

Still doesn't work
